### ISSUE TITLE
nitpick: *activities* buffer is in fundamental mode and has trailing white space

### ISSUE THREAD

**Issue by sleepyMonad** [alphapapa/activities.el#113]
I have 'show-trailing-whitespace' on globally, and turn it off for buffers where it doesn't matter with the buffer's mode hook. Fundamental mode however has no hooks by design. As a consequence the trailing space is rendered in my this-is-very-wrong typeface. My obsessive-compulsive side, I know...

PS: thanks for this package. I installed it an hour ago and already like it!

**Comment by alphapapa** [comment]
Sorry, but I don't think this is a problem that needs to be fixed.  This is an issue with user configuration.

I'm glad you like the package.

**Comment by sleepyMonad** [comment]
I have been looking in the code. Now I see that it even can't be fixed, the perceived "problem" is in vtable... and that code, _by design_, needs to pad its rendering to a fixed width. I also noticed that vtable has its own hook, solving my issue, not only for activities but also covering most of my list of modes where I want to ignore trailing white space.

Thank you for your feedback, I learned a few things.
